ctlra/meson.build

```diff
@@ -6,16 +6,27 @@ conf_data.set('jack', jack.found())
 
 subdir('devices')
 
+cargs = ['-Wno-unused-variable']
```

Comment:
You can do something like: `add_global_arguments('-Wno-unused-variable', language: 'c')` if you want it for everything and don't want to have to remember to add it to any new targets.

Reply:
Good point, yes will update that.